cmake/Re2cGenDocs.cmake

```diff
@@ -5,12 +5,13 @@ function(re2c_gen_manpage source target bootstrap lang)
         add_custom_command(
             OUTPUT "${target}"
             COMMAND "${CMAKE_COMMAND}" -E make_directory ${targetdir}
-            COMMAND "${re2c_splitman}" "${source}" "${source_l}" "${lang}"
-            COMMAND "${RST2MAN}" --tab-width=4 "${source_l}" "${target}"
+            COMMAND "${PYTHON}" "${re2c_splitman}" "${source}" "${source_l}" "${lang}"
+            COMMAND "${PYTHON}" "${re2c_rst2man}" --tab-width=4 "${source_l}" "${target}"
```

Comment:
Is `--tab-width=4` used in rst2man.py? I don't see the arguments either used or forwarded.

Reply:
Command-line parsing and handling is done by the Docutils `publish_cmdline` function. The `-tab-width` option gets forwarded to the rst parser.